Content.Shared/_DV/Holosign/ChargeHolosignSystem.cs

```diff
@@ -19,14 +19,13 @@ public sealed class ChargeHolosignSystem : EntitySystem
     [Dependency] private readonly SharedPopupSystem _popup = default!;
     [Dependency] private readonly SharedTransformSystem _transform = default!;
 
-    private HashSet<Entity<IComponent>> _signs = new();
+    private HashSet<Entity<IComponent>> _placedSigns = new();
```

Comment:
Make field readonly
```suggestion
    private readonly HashSet<Entity<IComponent>> _placedSigns = new();
```

Separately: `_container` is unused and can be removed
`using Robust.Shared.Map;` is unused and can be removed